_pages/team.html

```diff
@@ -10,232 +10,232 @@
             <p class="label">⭐️ Team Captain ⭐️</p>
-            <h5>Bobby Miller '22</h5>
-            <p>Bobby is the team captain in his fourth year on the team.</p>
+            <h5>Brandon Werbel '23</h5>
+            <p>Brandon is a fourth year programmer and the Team Captain.</p>
         </div>
         <div>
-            <img src="/images/team/placeholder.png">
-            <p class="label">🔧 Mechanic 🔧</p>
-            <h5>Smridhi Madan '22</h5>
-            <p>Smridhi is a third year mechanic.</p>
+            <img src="/images/team/david-oregan.png">
+            <p class="label">🖥️ Programming Captain 🖥️</p>
+            <h5>David O'Regan '23</h5>
+            <p>David is a second year programmer and the Programming Captain.</p>
         </div>
         <div>
             <img src="/images/team/nikita-johnson.png">
-            <p class="label">🔌 Electrician 🔌</p>
+            <p class="label">🔌 Electrical Captain 🔌</p>
             <h5>Nikita Johnson '23</h5>
-            <p>Nikita is a first year electrician.</p>
-        </div>
-        <div>
-            <img src="/images/team/argyle-lindsay.png">
-            <p class="label">🔧 Mechanic 🔧</p>
-            <h5>Argyle Lindsay '23</h5>
-            <p>Argyle is a third year mechanic</p>
-        </div>
-        <div>
-            <img src="/images/team/evan-hom.png">
-            <p class="label">🔧 Mechanic 🔧</p>
-            <h5>Evan Hom '23</h5>
-            <p>Evan is a third year mechanic.</p>
-        </div>
-        <div>
-            <img src="/images/team/finn-johnson.png">
-            <p class="label">🔧 Mechanic 🔧</p>
-            <h5>Finn Johnson '23</h5>
-            <p>Finn is a first year mechanic.</p>
-        </div>
-        <div>
-            <img src="/images/team/rani-altarazi.png">
-            <p class="label">🔧 Mechanic 🔧</p>
-            <h5>Rani Altarazi '23</h5>
-            <p>Rani is a first year mechanic. </p>
+            <p>Nikita is a second year electrician and the Electrical Captain.</p>
         </div>
         <div>
             <img src="/images/team/thomas-gallaway.png">
-            <p class="label">🔧 Mechanic 🔧</p>
+            <p class="label">🔧 Mechanics Captain 🔧</p>
             <h5>Thomas Gallaway '23</h5>
-            <p>"Just ziptie it!" Thomas is a third year mechanic.</p>
+            <p>"Just ziptie it!" Thomas is a fourth year mechanic and the Mechanics Captain.</p>
+        </div>
+        <div>
+            <img src="/images/team/sophia-gavasheli.png">
+            <p class="label">📸 Media 📸</p>
+            <h5>Sophia Gavasheli '23</h5>
+            <p>Sophia is a second year programmer.</p>
         </div>
         <div>
             <img src="/images/team/alexander-perez.png">
             <p class="label">💨 Pneumatics 💨</p>
             <h5>Alexander Perez '23</h5>
-            <p>Alexander is a second year pneumaticaneer.</p>
+            <p>Alexander is a third year pneumaticaneer.</p>
```

Comment:
pneumaticist?

Reply:
'pneumaticaneer' was the word used on the website previously, so I kept it for continuity. Also a bonus is that when searching 'pneumaticaneer' we are the only result lol

Reply:
lol